### ISSUE TITLE
Support of larger S3 file

### ISSUE THREAD

**Issue by StevenYongBGT** [blackflux/s3-cached#905]
When we use getBinaryObjectCached on a file size bigger than maybe 60MB, we hit an error:

FATAL ERROR: invalid table size Allocation failed - JavaScript heap out of memory

Does this support larger files on S3? We use it in a nodejs module and both Lambda, they are giving the same error.

**Update**: It's around 100MB, this will crash.

**Comment by simlu** [comment]
@StevenYongBGT I guess the next step is to check if we get the same error when just requesting the file from s3 without using this library. I can take a look at that tonight...

**Comment by simlu** [comment]
@StevenYongBGT Follow up: So the problem is the diskCache. Removing the diskcache allows for ~500mb files to be cached. I'm not really sure what the reason is, but I've tried all three different implementations `node-cache-manager-fs`, `node-cache-manager-fs-binary` and `node-cache-manager-fs-hash` and they all crash at around ~100mb (fs-hash was the best, but still very bad).

My previous statement stands and I would recommend that you go down a different avenue if you're dealing with files of this size. Unfortunately because of limitations that are out of the scope of this library only files < 100mb can be cached here.

Closing this, but feel free to follow up here!
